.github/workflows/integration_tests_validation.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        rust: [1.56.1, 1.57.0]
+        rust: [1.56.1, 1.58.0]
```

Comment:
Should this be 
```suggestion
        rust: [1.57.0, 1.58.0]
```

Reply:
Sorry for not explaining, here I want to run CI with min and max to support.